distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr.adoc

```diff
@@ -60,16 +60,15 @@ file `${home_directory}/etc/custom.system.properties`
 [NOTE]
 ====
 If ${branding} has generated an encrypted password, the password must be reset before
-proceeding. Follow the procedure in <<_restore_the_default_password_in_solr,
-Restore the Default Password in Solr>> before proceeding.
+proceeding.
 ====
 
 . To change the Solr password to a specific string, send Solr an HTTP POST request.
 This is covered in the official http://lucene.apache.org/solr/resources.html[Solr documentation]
  {external-link}. Here is an example that uses the
 command line utility `curl` to change the password from `admin` to `newpassword`:
 
- curl -k -u "admin:admin" "${secure_url}/solr/admin/authentication" -H 'Content-type:application/json' -d "{ 'set-user': {'admin' : 'newpassword'}}"
+ curl -k -u "admin:admin" "https://{FQDN}:{PORT}/solr/admin/authentication" -H 'Content-type:application/json' -d "{ 'set-user': {'admin' : 'newpassword'}}"
```

Comment:
Does Solr's basic auth plugin work with SolrCloud. I have never given it any thought until just now.

Reply:
It is possible.  SolrCloud just stores the security json file in Zookeeper instead.

https://lucene.apache.org/solr/guide/7_7/basic-authentication-plugin.html#basic-authentication-plugin
https://lucidworks.com/post/beyond-firewalls-how-basic-auth-can-protect-your-solrcloud/